docs/command/mongocli-config-delete.txt

```diff
@@ -57,3 +76,11 @@ Inherited Options
      - false
      - Profile to use from your configuration file.
 
+Examples
+--------
+
+.. code-block::
+
+ Delete a profile called myProfile:
+ $ mongocli config delete myProfile
```

Comment:
@gssbzn One more.. The examples should be indented two more spaces under .. code-block::. I submitted a PR to cobra2snooty which I think fixes it?
mongodb-labs/cobra2snooty#5

Reply:
so this is a bit of a conundrum, the reason we weren't spacing on `cobra2snooty` is because we were spacing on the examples already, for example on the `config command if I ask for help I'll get
```bash
mongocli config --help
Configure settings in a user profile.
All settings are optional. You can specify settings individually by running:
$ mongocli config set --help

You can also use environment variables (MCLI_*) when running the tool.
To find out more, see the documentation: https://docs.mongodb.com/mongocli/stable/configure/environment-variables/.

Usage:
  mongocli config [flags]
  mongocli config [command]

Examples:

  To configure the tool to work with Atlas
  $ mongocli config

  To configure the tool to work with Atlas for Government
  $ mongocli config --service cloudgov

  To configure the tool to work with Cloud Manager
  $ mongocli config --service cloud-manager

  To configure the tool to work with Ops Manager
  $ mongocli config --service ops-manager
```

with your changes we'll get

```bash
Usage:
  mongocli config [flags]
  mongocli config [command]

Examples:

Configure a profile to interact with Atlas:
$ mongocli config

Configure a profile to interact with Atlas for Government:
$ mongocli config --service cloudgov

Configure a profile to interact with Cloud Manager:
$ mongocli config --service cloud-manager

Configure a profile to interact with Ops Manager:
$ mongocli config --service ops-manager
```

from the cli side I'd like to keep some spacing there

```bash
Usage:
  mongocli config [flags]
  mongocli config [command]

Examples:
  Configure a profile to interact with Atlas:
  $ mongocli config

  Configure a profile to interact with Atlas for Government:
  $ mongocli config --service cloudgov

  Configure a profile to interact with Cloud Manager:
  $ mongocli config --service cloud-manager

  Configure a profile to interact with Ops Manager:
  $ mongocli config --service ops-manager
```

Reply:
Ah, I didn't realize this. So if we keep the 2 spaces, then the help page looks right and the RST is valid, I understand now. I'll put the spaces back and close the other PR.